src/components/Button/Button.scss

```diff
@@ -124,6 +124,11 @@
   border: none;
   vertical-align: top;
   line-height: normal;
+  font-size: 0;
```

Comment:
There seem to be some [potential issues with this method](https://css-tricks.com/fighting-the-space-between-inline-block-elements/#article-header-id-3).